app/views/tasks/_form.html.erb

```diff
@@ -13,19 +13,23 @@
 
   <div class="row">
     <div class="form-group col-md-4">
-      <%= f.label :task_name %><br>
+      <%= f.label :task_name %>
+      <button class="fit_to_row" type="button" data-toggle="tooltip" data-placement="right" title=" This name will appear in your task list and serve as an identification for the task. Consider making this descriptive for later reference. ">info</button><br>
```

Comment:
Bootstrap's buttons come with a lot of baggage (as you seem to have encountered). Instead of making these buttons, what about span tags? That way you won't have to fight bootstrap's styling. However, to get the pointer cursor (which adds to the UX, in my opinion), we could make a class for these tooltips that contains `cursor: pointer;`.

Also, since it doesn't appear to benefit accessibility at all (thanks, Bootstrap), let's try changing that 'info' text into a fun glyphicon! The one I had in mind was `glyphicon-question-sign`. For an example of using glyphicons, there's one further down in this page (the calendar).

P.S.S. There is a space before "This" and after the period. Please remove both.